mobile/src/lib/background.js

```diff
@@ -0,0 +1,72 @@
+import { getStore } from './store'
+import { initService } from './init'
+import { logException } from './crash-reporter'
+import { loadState } from './localStorage'
+import { startMediaUpload, mediaBackup, endMediaUpload } from '../actions/mediaBackup'
+import { backupAllowed } from './network'
+import { initPolyglot } from '../../../src/lib/I18n'
+
+const hasIosCordovaPlugin = () => {
+  return window.cordova !== undefined && window.cordova.platformId === 'ios' && window.BackgroundFetch !== undefined
+}
+
+export const startBackgroundService = () => {
+  if (hasIosCordovaPlugin()) {
+    startIosBackgroundService()
+  }
```

Comment:
We shall add and `else` with a `console.warn` instruction to prevent developers why the code does not seem to do anything.